ssl/statem/statem_srvr.c

```diff
@@ -1716,7 +1743,11 @@ static int tls_early_post_process_client_hello(SSL *s, int *pal)
         }
     }
 
-    if (!s->hit && s->version >= TLS1_VERSION && s->ext.session_secret_cb) {
+    if (!s->hit
+            && s->version >= TLS1_VERSION
+            && !SSL_IS_TLS13(s)
+            && !SSL_IS_DTLS(s)
+            && s->ext.session_secret_cb) {
```

Comment:
Conveniently (?), SSL_set_session_secret_cb() is undocumented, so there is no documentation to update to indicate that it cannot be used with TLS 1.3.
(I will not hold this work hostage to writing such a man page.)